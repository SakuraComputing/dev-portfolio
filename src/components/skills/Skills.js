import React from 'react';

const Skills = () => (

        <div className="item">
            <h3 className="level-title">Python &amp; Django<span className="level-label"
                                                                 data-toggle="tooltip" data-placement="left"
                                                                 data-animation="true"
                                                                 title="You can use the tooltip to add more info...">Expert</span>
            </h3>
            <div className="level-bar">
                <div className="level-bar-inner" data-level="96%">
                </div>
            </div>
        </div>
);

export default Skills;