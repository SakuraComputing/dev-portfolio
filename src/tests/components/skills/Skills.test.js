import React from 'react';
import { shallow } from 'enzyme';
import Skills from '../../../components/skills/Skills';

describe('Skills', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<Skills />);

    });
   it('should render the for correctly', () => {
       expect(wrapper).toMatchSnapshot()
    })
});