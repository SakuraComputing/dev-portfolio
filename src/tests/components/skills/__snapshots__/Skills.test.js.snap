// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills should render the for correctly 1`] = `
<div
  className="item"
>
  <h3
    className="level-title"
  >
    Python & Django
    <span
      className="level-label"
      data-animation="true"
      data-placement="left"
      data-toggle="tooltip"
      title="You can use the tooltip to add more info..."
    >
      Expert
    </span>
  </h3>
  <div
    className="level-bar"
  >
    <div
      className="level-bar-inner"
      data-level="96%"
    />
  </div>
</div>
`;
